client/state/themes/selectors.js

```diff
@@ -65,6 +65,11 @@ export const getTheme = createSelector(
 		if ( siteId === 'wpcom' || siteId === 'wporg' ) {
 			return theme;
 		}
+
+		if ( ! theme ) {
+			return null;
```

Comment:
This is not strictly necessary for this PR, but it does prevent the merge of an empty result with the wporg fields below.